cmd/agent/gui/views/templates/generalStatus.tmpl

```diff
@@ -183,9 +183,12 @@
               {{- range $check,$instances := .checks.initialized_checks }}
                 {{ $check -}}<br>
                 {{- range $instance := $instances }}
-                  {{- range $k,$v := $instance }}
-                      {{ $k }} : {{ $v }}<br>
-                  {{- end -}}
+                    instance_name: {{ .instance_name }}<br>
+                    metric_count: {{ .metric_count }}<br>
+                    service_check_count: {{ .service_check_count }}<br>
+                    message: {{ .message }}<br>
+                    status: {{ .status }}<br>
+                    <br>
```

Comment:
Since this is reused in GUI + status and we explicitly name the fields, it could be useful to have this snippet in a separate shared template but it's not a blocker for the PR